java/ql/test-kotlin2/library-tests/controlflow/basic/bbStmts.expected

```diff
@@ -149,27 +150,27 @@
 | Test.kt:101:33:103:5 | { ... } | 2 | Test.kt:102:9:102:25 | throw ... |
 | Test.kt:105:5:109:5 | <Expr>; | 0 | Test.kt:105:5:109:5 | <Expr>; |
 | Test.kt:105:5:109:5 | <Expr>; | 1 | Test.kt:105:5:109:5 | when ... |
-| Test.kt:105:5:109:5 | <Expr>; | 2 | Test.kt:105:9:107:5 | ... -> ... |
+| Test.kt:105:5:109:5 | <Expr>; | 2 | Test.kt:105:9:105:17 | ... -> ... |
 | Test.kt:105:5:109:5 | <Expr>; | 3 | Test.kt:105:9:105:9 | x |
 | Test.kt:105:5:109:5 | <Expr>; | 4 | Test.kt:105:14:105:17 | null |
 | Test.kt:105:5:109:5 | <Expr>; | 5 | Test.kt:105:9:105:17 | ... (value not-equals) ... |
 | Test.kt:105:20:107:5 | { ... } | 0 | Test.kt:105:20:107:5 | { ... } |
 | Test.kt:105:20:107:5 | { ... } | 1 | Test.kt:106:9:106:29 | <Expr>; |
-| Test.kt:105:20:107:5 | { ... } | 2 | Test.kt:106:18:106:27 | "x not null" |
+| Test.kt:105:20:107:5 | { ... } | 2 | Test.kt:106:17:106:28 | "x not null" |
 | Test.kt:105:20:107:5 | { ... } | 3 | Test.kt:106:9:106:29 | println(...) |
-| Test.kt:107:16:109:5 | ... -> ... | 0 | Test.kt:107:16:109:5 | ... -> ... |
-| Test.kt:107:16:109:5 | ... -> ... | 1 | Test.kt:107:16:107:16 | y |
-| Test.kt:107:16:109:5 | ... -> ... | 2 | Test.kt:107:21:107:24 | null |
-| Test.kt:107:16:109:5 | ... -> ... | 3 | Test.kt:107:16:107:24 | ... (value not-equals) ... |
+| Test.kt:107:16:107:24 | ... -> ... | 0 | Test.kt:107:16:107:24 | ... -> ... |
+| Test.kt:107:16:107:24 | ... -> ... | 1 | Test.kt:107:16:107:16 | y |
+| Test.kt:107:16:107:24 | ... -> ... | 2 | Test.kt:107:21:107:24 | null |
+| Test.kt:107:16:107:24 | ... -> ... | 3 | Test.kt:107:16:107:24 | ... (value not-equals) ... |
 | Test.kt:107:27:109:5 | { ... } | 0 | Test.kt:107:27:109:5 | { ... } |
 | Test.kt:107:27:109:5 | { ... } | 1 | Test.kt:108:9:108:29 | <Expr>; |
-| Test.kt:107:27:109:5 | { ... } | 2 | Test.kt:108:18:108:27 | "y not null" |
+| Test.kt:107:27:109:5 | { ... } | 2 | Test.kt:108:17:108:28 | "y not null" |
 | Test.kt:107:27:109:5 | { ... } | 3 | Test.kt:108:9:108:29 | println(...) |
 | Test.kt:112:1:116:1 | fn | 0 | Test.kt:112:1:116:1 | fn |
 | Test.kt:112:32:116:1 | { ... } | 0 | Test.kt:112:32:116:1 | { ... } |
 | Test.kt:112:32:116:1 | { ... } | 1 | Test.kt:113:5:115:5 | <Expr>; |
 | Test.kt:112:32:116:1 | { ... } | 2 | Test.kt:113:5:115:5 | when ... |
-| Test.kt:112:32:116:1 | { ... } | 3 | Test.kt:113:5:115:5 | ... -> ... |
+| Test.kt:112:32:116:1 | { ... } | 3 | Test.kt:113:9:113:14 | ... -> ... |
 | Test.kt:112:32:116:1 | { ... } | 4 | Test.kt:113:9:113:14 | ... && ... |
 | Test.kt:112:32:116:1 | { ... } | 5 | Test.kt:113:9:113:9 | x |
 | Test.kt:113:14:113:14 | y | 0 | Test.kt:113:14:113:14 | y |
```

Comment:
This file got 1 line longer. It looks like there's not only location change in the file.

Reply:
Ah, I should have said: The "Accept loc changes" commits only contain location changes, but the "Accept changes" commits also accept other changes. In this case, there is an implicit `{}` added around `y = 40` in `if(x < 0) y = 40`, so we have 1 additional statement.